src/Slack.py

```diff
@@ -38,13 +38,25 @@
 sc = SlackClient(SlackApiKey)
 sc.rtm_connect()
 
-# Set up URL's and headers for Discord.
+# Set up URL"s and headers for Discord.
```

Comment:
URL's not URL"s

Reply:
yeah autoreplacing, also we should change " to ' in Discord.py too

Reply:
For comments no. But Sure.